src/coveralls.erl

```diff
@@ -381,8 +363,13 @@ realpath_and_relative_test_() ->
   {setup,
    fun() -> %% setup
        {ok, Cwd} = file:get_cwd(),
-       Root = string:strip(
-                os:cmd("mktemp -d -t coveralls_tests.XXX"), right, $\n),
+       RandNum = case erlang:function_exported(rand, uniform, 1) of
```

Comment:
this seems completely unrelated to source digest, why the change?

Reply:
I'd prefer if that was part of a separate PR and that this PR would be squashed into 1 commit :).